src/lib/storage/segment_access_statistics.cpp

```diff
@@ -0,0 +1,171 @@
+#include "segment_access_statistics.hpp"
+
+namespace opossum {
+
+std::chrono::time_point<std::chrono::steady_clock> AtomicTimedAccessStrategy::start_time = std::chrono::steady_clock::now();
+std::chrono::duration<double, std::milli> AtomicTimedAccessStrategy::interval = std::chrono::milliseconds{1000};
+
+// -------------------------------------------------------------------------------------------------------------------
+SegmentAccessType SegmentAccessTypeTools::iterator_access_pattern(const std::shared_ptr<const PosList>& positions) {
```

Comment:
Scannt die ersten 100 Elemente in positions, um das Zugriffsmuster _sequentiell_ (0,0,1,2,2,3), _aufsteigend_ (0,0,1,2,2,4) oder _zufällig_ (0,0,1,2,2,1) zu ermitteln.